core-services/src/main/java/org/zalando/nakadi/service/publishing/NakadiKpiPublisher.java

```diff
@@ -49,6 +53,42 @@ protected NakadiKpiPublisher(
         this.eventMetadata = eventMetadata;
         this.uuidGenerator = uuidGenerator;
         this.avroSchema = avroSchema;
+        this.kpiEventMapper = new KPIEventMapper(Set.of(SubscriptionLogEvent.class));
+    }
+
+    public void publish(final Supplier<KPIEvent> kpiEventSupplier) {
```

Comment:
I miss usage of the method

Reply:
Good point; forgot to use the new feature.

Reply:
This is done now.